### PR TITLE
Add Cifar 100 C

### PR DESCRIPTION
**Pull request by lsrosa**
Load CIFAR-100-C.

Add some docs.

### CODE REVIEW

File: peepholelib/utils/analyze.py [hunk 1]
```diff
@@ -27,99 +30,114 @@ def compute_top_k_accuracy(peepholes, targets, k):
...
+        'score': {},
+        'protoclasses': proto,
     }
 
+    # for normalization
+    _line = torch.zeros(nc)
+    _line[0] = 1
+    min_e = Categorical(probs=_line).entropy()
+    _unif = torch.ones(nc)/nc
+    max_e = Categorical(probs=_unif).entropy()
```
Comment (lorenzoCapelli) [inline peepholelib/utils/analyze.py:103]: shouldn't these lines be eliminated since we are working on protoclasses

File: peepholelib/utils/analyze.py [hunk 2]
```diff
@@ -182,63 +202,111 @@ def conceptogram_entropy_score(**kwargs):
...
     target_modules = kwargs.get('target_modules', None)
     proto_key = kwargs.get('proto_key', 'train')
+    bins = kwargs.get('bins', 20)
     plot = kwargs.get('plot', False)
-    drop_max = kwargs.get('max_drop', 20)
     verbose = kwargs.get('verbose', False)
-    
-    score_name = 'Proto-Class'
+    atk_name = kwargs.get('atk_name', None)
+    path = kwargs.get('path')
```
Comment (lorenzoCapelli) [inline peepholelib/utils/analyze.py:287]: proto_th = kwargs.get('proto_threshold', 0.9) proto_th is missing in this case

File: peepholelib/utils/analyze.py [hunk 3]
```diff
@@ -248,114 +316,85 @@ def conceptogram_protoclass_score(**kwargs):
...
-        # main computation
-        _wcps = (proto[labels]*cps).sum(dim=1)
-
-        # normalization does not matter for entropy
         s = (proto[labels]*cps).sum(dim=(1,2))
-        scores = s/(torch.norm(proto[labels], dim=(1,2))*torch.norm(cps, dim=(1,2)))
-
-        # compute AUC for score
-        s_auc = AUC().update(scores, results.int()).compute().item()
+        so = s/(torch.norm(proto[labels], dim=(1,2))*torch.norm(cps, dim=(1,2)))
```
Comment (lorenzoCapelli) [inline peepholelib/utils/analyze.py:331]: also here we look at the label but it is incorrect

File: peepholelib/utils/analyze.py (hunk 1 above)
Comment (lsrosa) [reply peepholelib/utils/analyze.py:103]: those will be cleaned with the scores rework
